src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -119,6 +120,7 @@ private void UnsubscribeFromDynamoEvents()
             LoadedParams.CurrentWorkspaceChanged -= OnCurrentWorkspaceChanged;
             DynamoViewModel.CurrentSpaceViewModel.Model.NodeAdded -= OnNodeAdded;
             DynamoViewModel.Model.Logger.NotificationLogged -= OnNotificationLogged;
+            //CurrentWorkspace.RequestPackageDependencies -= PythonDependencies.AddPythonPackageDependency;
```

Comment:
why remove this and the other unsubscribe statements? It's hard to reason about if the extra handlers will cause memory leaks.

Reply:
sorry, missed your commit.